setup.py

```diff
@@ -13,6 +15,26 @@ def convert(filename, fmt):
         with io.open(filename, encoding='utf-8') as fd:
             return fd.read()
 
+def create_mo_files():
+    data_files = []
+    localedir = 'cmsplugin_cascade/locale'
+    po_dirs = [localedir + '/' + l + '/LC_MESSAGES/'
+               for l in next(os.walk(localedir))[1]]
+    for d in po_dirs:
+        mo_files = []
+        po_files = [f
+                    for f in next(os.walk(d))[2]
+                    if os.path.splitext(f)[1] == '.po']
+        for po_file in po_files:
+            filename, extension = os.path.splitext(po_file)
+            mo_file = filename + '.mo'
+            msgfmt_cmd = 'msgfmt {} -o {}'.format(d + po_file, d + mo_file)
```

Comment:
why don't you use the ``django-admin`` command. Can even be invoked without subprocess.

Reply:
I understood now  after installation, to have the translations, we need to run the command "django-admin compilemessages"